src/analysis/retail/monk/mistweaver/CombatLogParser.ts

```diff
@@ -52,7 +52,6 @@ import AverageTimeBetweenRSKSs from './modules/spells/AverageTimeBetweenRSKs';
 import ChiBurst from './modules/spells/ChiBurst';
 import InvokeChiJi from './modules/spells/InvokeChiJi';
 import JadeSerpentStatue from './modules/spells/JadeSerpentStatue';
-//import Lifecycles from './modules/spells/Lifecycles';
 import ManaTea from './modules/spells/ManaTea';
```

Comment:
2 ways to go about this:

1.  We have more info that we should detail out when talenting lifecycles - We can either keep the lifecycles stub out and rework it shortly here or
2. We can add the additional LC details to this module. 

I personally like the idea of leaving this as is and keeping a new LC module. This one works well as a supplemental module for both talents  in a non-talent category like Core or Threorycraft.  
LMK what you think

Reply:
I think new Lifecycles module is probably cleanest since this is already 2 pie charts.